### DISCUSSION TITLE
children.join is not a function

### DISCUSSION THREAD

**Discussion by aakash1419** [vercel/next.js#16814] [Help]
I am getting a strange error on only one page in my nextjs project...other pages work fine i.e `children.join is not a function`.

it seems to be different from #2175 & #910

here's the screenshot of the error

![image](https://user-images.githubusercontent.com/49414229/92101336-9a9feb00-edfa-11ea-8b6a-31bb39ed207a.png)

**Comment by jamesmosier** [comment]
Do you happen to have some code you could share?

Also based on that stack trace, [the next/head docs](https://nextjs.org/docs/api-reference/next/head) mention this:

> title, meta or any other elements (e.g. script) need to be contained as direct children of the Head element, or wrapped into maximum one level of <React.Fragment> or arrays—otherwise the tags won't be correctly picked up on client-side navigations.

so if you are using next/head make sure that all element are direct children of `<Head>`

**Reply by nghiepnguyen520** [reply]
It did not work for me
